src/lib.rs

```diff
@@ -87,20 +89,25 @@ pub mod vc {
     pub use super::iter::*;
 }
 
+#[cfg(any(test, miri))]
+pub type Vc<T> = RawVc<T, 4>;
+#[cfg(not(any(test, miri)))]
+pub type Vc<T> = RawVc<T, 8>;
+
 /// A `VecDeque` (and `Vec`) variant that spreads resize load across pushes.
 ///
 /// See the [crate-level documentation] for details.
 ///
 /// [crate-level documentation]: index.html
-pub struct Vc<T> {
+pub struct RawVc<T, const R: usize> {
```

Comment:
I'm torn on the naming here. I see why you chose `RawVc`, but at the same time "raw" tends to imply unsafe/low-level, which this isn't really. Not sure what's a better name that isn't also way longer though. Maybe `VcC`?

I also think we need to add documentation to the `Vc` type alias saying "this is `RawVc` with the `const` set to a reasonable default value"?

Reply:
I feel like `VcC` would be a bit confusing/easy to confuse with normal Vc, so a more distinguished name feels appropriate. Maybe `CustomVc` or something like that?

The Docs in general need a bit more work now again, both on the type alias as well as the [`move_amount`](https://github.com/jonhoo/atone/blob/79a65402b8dae6ff484b69042162078811df3bc2/src/lib.rs#L146C7-L146C7) function, which i am also not sure on the name

Reply:
Let's go with `CustomVc` :+1: